# 状态量
SingleObjectState[] as_agent_states # ego
SingleObjectState[] as_goal_states  # goal
SingleObjectState[] os_obst_states  # obstacles
int32 state_dim # state_dim=8，方便解析
